lib/updatehub/device.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DEVICE_H__
+#define __DEVICE_H__
+
+#include <zephyr.h>
+#include <hwinfo.h>
+
+#define DEVICE_ID_MAX_SIZE 65
+
+static inline bool updatehub_get_device_identity(char *id, int id_max_len)
```

Comment:
Is there a reason this is in a header?

Reply:
Because that function is used on shell.c and the updatehub.c.

Reply:
Is there any reason to inline this, though? Why not just have a prototype in this file, and put the function body in a C file. The function call overhead is very small compared to what the function is doing, and inlining will only waste space.

Reply:
I'm asking why in a .h vs .c?

Reply:
Okay, I understood now.